webpack.config.base.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
 
+var _ = require('lodash');
+var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
   module: {
-    loaders: [
-      { test: /\.js$/, loaders: ['babel-loader'], exclude: /node_modules/ }
-    ]
+    loaders: [{
+      test: /\.js$/,
+      loaders: ['babel-loader'],
+      include: [
+        path.resolve(__dirname, 'src'),
+        path.resolve(__dirname, 'node_modules', 'lodash')
+      ]
+    }]
+  },
+  node: {
+    process: false
   },
   output: {
     library: 'Redux',
     libraryTarget: 'umd'
   },
-  resolve: {
-    extensions: ['', '.js']
-  }
+  plugins: [
+    ['lodash/internal/baseIteratee.js', './toFunction.js'],
+    ['lodash/object/keys.js',           '../internal/baseKeys.js'],
```

Comment:
Replacing `keys` and `keysIn` with their base forms since yall aren't concerned with treating sparse arrays as dense or compat paths for older enviros.